qemu/tests/cdrom.py

```diff
@@ -600,8 +600,8 @@ def test(self):
             error.context("Detecting the existence of a cdrom (qemu side)",
                           logging.info)
             qemu_cdrom_device = get_device(vm, iso_image)
-
-            self.session.get_command_output("umount %s" % guest_cdrom_device)
```

Comment:
1. could you check why need to umount
2. cmd to windows: "mountvol %s /d", it's better to make it configurable in cfg file if the step is needed
3. Suggest to get winutils drive with utils_misc.get_winutils_vol(session) instead of hard code it.